### DISCUSSION TITLE
Better support for ignoredPaths in onboarding PR

### DISCUSSION THREAD

**Discussion by TWiStErRob** [renovatebot/renovate#17379] [Suggest an Idea]
### What would you like Renovate to be able to do?

Have a look at https://github.com/TWiStErRob/net.twisterrob.gradle/pull/228 (screenshot here for future reference)
<img src=https://user-images.githubusercontent.com/2906988/186366891-fb61a5db-9243-4c35-80ce-e72994f52d8e.png width=10% />

Notice that I edited the config multiple times. Also notice the amount of paths it detects.

Now consider that I added this to the config:
```json
	"ignorePaths": [
		"/docs/debug/**",
		"/docs/examples/**",
		"/graph/**",
		"/**/src/test/resources/**"
	],
```
and yet the list is still very long and the same. This feature request is about flagging somehow what these glob patterns resolve, 

### If you have any ideas on how this should be implemented, please tell us here.

An option is listing ~`ignored/paths` as striked through~ (`~` in markdown)
![image](https://user-images.githubusercontent.com/2906988/186367694-65b95b3d-37bd-4ef9-95eb-22f49a1b97dc.png)

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
This is not possible with our current approach to this feature because files matching ignoredPaths are removed from the branch file list prior to package files being detected. In other words we don't know which package files are ignored so cannot list them in onboarding.

Second, even if files aren't ignored, just because they match a `fileMatch` pattern doesn't mean they would make the list. Renovate discards matching files which contain no dependencies, especially for managers which might e.g. match against all `*.yaml` files when looking for k8s files.

**Reply by gberche-orange** [reply]
> and yet the list is still very long and the same. 

On my side, I do see the `ignorePath` being taken into account and the onboarding PR being updated with ignored paths being removed from the list
![image](https://user-images.githubusercontent.com/4748380/186451845-e8b95314-c64f-4446-918e-972f4b52ebf3.png)

**Reply by TWiStErRob** [reply]
@gberche-orange where do you see the changes taken into account? please open the diff of the edit and notice that there're no changes to file paths.

**Reply by gberche-orange** [reply]
@TWiStErRob I'm indeed observing changes to detected package files

![image](https://user-images.githubusercontent.com/4748380/186483625-bbdef604-8e2e-4650-9336-e1565255ede1.png)

**Reply by TWiStErRob** [reply]
That's not my PR. In one version I see added files, because I updated from master, but when I added "ignoredPaths" nothing has changed in the list. So I must be using the wrong syntax?

**Reply by TWiStErRob** [reply]
Can you share a live example of ignorePaths?

**Reply by TWiStErRob** [reply]
ah, ok found one that's similar to my use case https://github.com/spectreconsole/spectre.console/blob/0d19ccd8a6236d464f1cb397df7f55554d30e0d7/.github/renovate.json#L16-L19
and now it's hidden: https://github.com/TWiStErRob/net.twisterrob.gradle/pull/228/commits/67a672414c1b8879ec645f21f152fb37657e5804

**Reply by rarkins** [reply]
Sorry, I didn't realise that your ignorePaths "wasn't working"

**Reply by TWiStErRob** [reply]
Me neither :D

**Reply by TWiStErRob** [reply]
@rarkins would it make sense to fail or at least warn on ignores starting with `/`? It seems they're relative to project root, but the absolute path is probably absolute from unix root.
